### PR TITLE
Gitbooks

### PR DESCRIPTION
**Pull request by sbsmith86**
#### What's this PR do?

The goal of this PR is to get us started using [Gitbook](https://www.gitbook.com/). 

It mainly renames the `documentation` directory to `docs` which Gitbooks looks for and then adds a `Summary.md` that gitbooks uses to map out the table of contents. 

image

#### How should this be reviewed?

👀 

#### Any background context you want to provide?

The goal here wasn't to get everything fully onto gitbooks right now, but to start to set up the structure to build on this iteratively. 

I think we should aim to get everything that is currently on our WIKI to gitbook. Also I would love for us to move finalized documentation to gitbook instead of keeping thing in google docs, so that it is easily searchable. 

Also more to come about how to use this. 

#### Relevant tickets

:shae-hack-day-fun:

### CODE REVIEW

**Comment by DFurnes** [comment]
Excited about this!! 🎉

File: docs/development/deployments.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Deployments
+
+We generally deploy Rogue every weekday morning at 10:30am. That being said, we encourage developers to be continously deploying changes to production throughout the day to keep deployments as small and as isolated as possible.
+
+The general rule of thumb is to do a qucik check with the developer team in the #rogue slack channel to confirm it is OK to deploy to production. If it is late in the day, a Friday, or there is some other blocking change, we might ask that we hold off on a production deployment.
```
Comment (chloealee) [inline docs/development/deployments.md]: shoot sorry @sbsmith86 ! Found a little typo "qucik"

File: docs/development/deployments.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# General deployment workflow
+
+When you submit a PR against the master branch in the [Rogue repository](https://github.com/DoSomething/rogue) a [Wercker](https://app.wercker.com/dosomething/rogue/runs) build will be triggered and deployed to a Review App on Heroku. You can use this Review app to share and test your specific changes. Once the PR is merged and closed, a deployment will be automatically triggered to the QA and staging environments. See the [Architecture](/docs/development/architecture.md) section for more information on the differences between environments.
+
+Once your code is tested on rogue-staging (a.k.a rogue-thor). You can trigger a deployment to Rogue production.
```
Comment (chloealee) [inline docs/development/deployments.md]: should the period after `rogue-thor)` be a `,` and then `You` be lowercased?

**Comment by chloealee** [comment]
a few nitpicky things but otherwise this is amazing @sbsmith86 ! Thanks so much for providing so much detail!!!

File: docs/development/contributing.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+This section will go over some general points about contributing to Rogue as a developer. If you have any specific questions that are not addressed here, please reach out to Team Bleed.
+
+## Pivotal Tracker
+We use [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2019429) as our project management tool for tracking all of our teams work sprint-by-sprint.
+
+Rogue maintenance and features are maintained primarily by Team Bleed. Please reach out to the product
+team if you are having issues accessing the Team Bleed pivotal board.
```
Comment (chloealee) [inline docs/development/contributing.md]: Should `pivotal` be capitalized? 😬

**Comment by katiecrane** [comment]
THIS IS SO EXCITING!!!!!! ✍️ 📖

File: README.md [hunk 4]
```diff
@@ -2,51 +2,21 @@
...
-# And finally, build the frontend assets:
-$ npm run build
-```
+Maintained by [Team Bleed](https://github.com/orgs/DoSomething/teams/team-bleed)
+- [Jen](https://github.com/ngjo) - Product Manager
+- [Shae](https://github.com/sbsmith86) - Tech Lead
+- [Katie](https://github.com/katiecrane) - Engineer
+- [Chloe](https://github.com/chloealee) - Engineer
+- [Luke](https://github.com/lkpttn) - Product Designer
+- [Dave](https://github.com/DFurnes) - Staff Engineer
```
Comment (weerd) [inline README.md]: Good idea with this! Gonna add this to the Phoenix read me as well 💃

File: package.json [hunk 5]
```diff
@@ -8,7 +8,9 @@
     "test": "jest --watch",
     "test:ci": "jest",
     "heroku-postbuild": "npm run build",
-    "lint": "eslint --fix --ext .js resources/assets"
+    "lint": "eslint --fix --ext .js resources/assets",
+    "docs:prepare": "gitbook install",
+    "docs:watch": "npm run docs:prepare && gitbook serve"
```
Comment (weerd) [inline package.json]: Oh nice! I didn't think to add something like this in the Phoenix repo. I was thinking of keeping it separate but I like this. Good idea @sbsmith86 👍

Comment (sbsmith86) [reply package.json]: it's very nice for local dev!

**Pull request merged**